HospitalSystem.Core/Uitls/Globals.cs

```diff
@@ -3,8 +3,8 @@ namespace HospitalSystem.Core.Utils;
 
 public static class Globals
 {
-    public static DateTime OpeningTime { get; set; } = new DateTime(2000, 10, 20, 9, 0, 0);
-    public static DateTime ClosingTime { get; set; } = new DateTime(2000, 10, 20, 17, 0, 0);
+    public static DateTime OpeningTime { get; set; } = new DateTime(DateTime.Now.Year, DateTime.Now.Month, DateTime.Now.Day, 9, 0, 0);
+    public static DateTime ClosingTime { get; set; } = new DateTime(DateTime.Now.Year, DateTime.Now.Month, DateTime.Now.Day, 17, 0, 0);
```

Comment:
These opening times should not even have Dates (let alone being in globals, which is not a problem for you). Should keep just Time

Reply:
This is not my class, i just changed it for easier testgenerate